src/html/tree.ml

```diff
@@ -70,6 +68,34 @@ let page_creator ~config ~url name header toc content =
         Html.script ~a:[ Html.a_src highlight_js_uri ] (Html.txt "");
         Html.script (Html.txt "hljs.initHighlightingOnLoad();");
       ]
+    in
+    let meta_elements =
+      if uses_katex then
+        let katex_css_uri = file_uri theme_uri "katex.min.css" in
+        let katex_js_uri = file_uri support_uri "katex.min.js" in
+        default_meta_elements
+        @ [
+            Html.link ~rel:[ `Stylesheet ] ~href:katex_css_uri ();
+            Html.script ~a:[ Html.a_src katex_js_uri ] (Html.txt "");
+            Html.script
+              (Html.cdata_script
+                 {|
+          document.addEventListener("DOMContentLoaded", function () {
+            var elements = document.getElementsByClassName("odoc-katex-math");
+            for (var i = 0; i < elements.length; i++) {
+              var el = elements[i];
+              var content = el.textContent;
+              var new_el = document.createElement("span");
+              new_el.setAttribute("class", "odoc-katex-math-rendered");
+              katex.render(content, new_el, { throwOnError: false });
```

Comment:
We need to tell katex to render display style rather than inline explicitly. We could, for example, add a class to display elements, and then:

```suggestion
              var display = el.classList.contains("display");
              katex.render(content, new_el, { throwOnError: false, displayMode: display });
```